template/packages/frontend/components/RecipeStyles.ts

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components"
+
+export const Button = styled.button<{ primary?: boolean }>`
+  background: ${props =>
+    props.primary ? props.theme.colors.accent2 : props.theme.colors.white};
+  color: ${props =>
+    props.primary ? props.theme.colors.white : props.theme.colors.accent2};
+  font-size: 1em;
+  margin: 1em;
+  padding: 0.25em 1em;
+  border: 2px solid ${props => props.theme.colors.accent2};
+  border-radius: 3px;
+  width: 10rem;
+  :focus {
+    border-color: ${props => props.theme.colors.accent};
+    outline: none;
+  }
+  :hover {
+    cursor: pointer;
+  }
```

Comment:
Since I was messing with cursor shapes I added this. I think it's nicer if the cursor changes to a pointer when over a button.